massdriver-application-azure-function-app/main.tf

```diff
@@ -0,0 +1,88 @@
+locals {
+  max_length           = 24
+  storage_account_name = substr(replace(var.name, "/[^a-z0-9]/", ""), 0, local.max_length)
+}
+
+module "application" {
+  source                      = "github.com/massdriver-cloud/terraform-modules//massdriver-application?ref=22d422e"
+  name                        = var.name
+  service                     = "function"
+  application_identity_id     = azurerm_linux_function_app.main.identity[0].principal_id
+  create_application_identity = false
+}
+
+resource "azurerm_resource_group" "main" {
+  name     = var.name
+  location = var.application.location
+  tags     = var.tags
+}
+
+resource "azurerm_service_plan" "main" {
+  name                = var.name
+  resource_group_name = azurerm_resource_group.main.name
+  location            = azurerm_resource_group.main.location
+  os_type             = "Linux"
+  sku_name            = var.application.sku_name
+  tags                = var.tags
+}
+
+resource "azurerm_linux_function_app" "main" {
+  name                        = var.name
+  resource_group_name         = azurerm_resource_group.main.name
+  location                    = azurerm_resource_group.main.location
+  service_plan_id             = azurerm_service_plan.main.id
+  https_only                  = true
+  storage_account_name        = azurerm_storage_account.main.name
+  storage_account_access_key  = azurerm_storage_account.main.primary_access_key
+  functions_extension_version = "~4"
+  virtual_network_subnet_id   = azurerm_subnet.main.id
+  tags                        = var.tags
+
+
+  app_settings = merge({
+    FUNCTIONS_WORKER_RUNTIME = var.application.runtime
```

Comment:
What is the runtime here? Does the container not hold the runtime?

Reply:
Seems to work locally without this value. I'll remove it.